modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractPhpCodegen.java

```diff
@@ -68,7 +79,8 @@ public AbstractPhpCodegen() {
         apiPackage = invokerPackage + "\\" + apiDirName;
         modelPackage = invokerPackage + "\\" + modelDirName;
 
-        setReservedWordsLowerCase(
+        // TODO Are those case sensitive or insensitive
```

Comment:
in PHP, variable and class names are case insensitive

Reply:
Resolved in commit [11ff0928](https://github.com/OpenAPITools/openapi-generator/commit/11ff0928fd16a8a9342c43b65026e1c4dc973d3d)